cmd/otelcol/config.yaml

```diff
@@ -45,11 +51,11 @@ service:
     traces:
       receivers: [otlp, opencensus, jaeger, zipkin]
       processors: [batch]
-      exporters: [logging]
+      exporters: [otlphttp]
```

Comment:
Why is logging exporter no longer desired?

Reply:
Accidental change while testing. Will revert.

Reply:
Fixed.